apps/frontend/core-dapp/src/features/trade/close-trade/ClosePositionButton.tsx

```diff
@@ -32,10 +32,10 @@ const ClosePositionButton: React.FC = () => {
     if (
       closePositionValueBN !== undefined &&
       closePositionValueBN.gt(0) &&
```

Comment:
if there are both ETH and wstETH values in a function, it'd be clearer to indicate which ones are ETH vs wstETH
```suggestion
      closePositionValueInEthBN !== undefined &&
      closePositionValueInEthBN.gt(0) &&
```

Reply:
I wanted to minimize changes to existing code, I can take a pass at this on another PR